frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -149,6 +182,43 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
   const canViewProperty = keycloak.canUserViewProperty(propertyInfo);
   const canEditProperty = keycloak.canUserEditProperty(propertyInfo);
 
+  const renderContent = () => {
```

Comment:
I made a separate function to do the rendering, the conditionals were just getting ugly and confusing